setup.cfg

```diff
@@ -1,104 +0,0 @@
-[flake8]
-paths=packages
-exclude=.md,
-    *_pb2.py,
-    *_pb2_grpc.py,
-    custom_types.py,
-
-max-line-length = 88
-select = B,C,D,E,F,I,W,
-ignore = E203,E501,W503,D202,B014,D400,D401,DAR
-application-import-names = packages,tests,scripts
-
-# ignore as too restrictive for our needs:
-# D400: First line should end with a period
-# D401: First line should be in imperative mood
-# E501: https://www.flake8rules.com/rules/E501.html (Line too long)
-# E203: https://www.flake8rules.com/rules/E203.html (Whitespace)
-# W503: https://www.flake8rules.com/rules/W503.html (Line break)
-# D202: blank lines
-# B014: redundant exception
-
-[isort]
-# for black compatibility
-multi_line_output=3
-include_trailing_comma=True
-force_grid_wrap=0
-use_parentheses=True
-ensure_newline_before_comments = True
-line_length=88
-# custom configurations
-order_by_type=False
-case_sensitive=True
-lines_after_imports=2
-skip=packages/valory/skills/abstract_round_abci/test_tools/integration.py,packages/valory/skills/transaction_settlement_abci/test_tools/integration.py,
-skip_glob =
-known_first_party=autonomy
-known_packages=packages
-known_local_folder=tests
-sections=FUTURE,STDLIB,THIRDPARTY,FIRSTPARTY,PACKAGES,LOCALFOLDER
-
-[mypy]
-python_version = 3.8
-strict_optional = True
-exclude=(.*_pb2|.*custom_types)
-
-# Before adding a module here, make sure it does not support type hints
-# Per-module options for aea dir:
-
-[mypy-packages.valory.*]
-ignore_errors=True
```

Comment:
This was effectively ignoring everything so we now get a lot of `mypy` errors.

Reply:
Ignored everything again to get unblocked on 49b9b58c9406f7403d3c831f86a37dcb65e3a1cd.